### PR TITLE
chore(deps): bump dependencies

### PR DESCRIPTION
**Pull request by ttbarnes**
## Introduction :pencil2:
- Dependabot flagged some dependency updates.

## Resolution :heavy_check_mark:
- Update dependencies.
- Update relevant intergrity hashes.

### CODE REVIEW

File: src/ui/server/constants/integrity.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 export const INTEGRITY = {
   JS: 'sha512-BFOXngFJ+9msC1y5Q1LD6AG+LG635vp8Lfv/+HVCUU8w/N8VWH13wdTyZeLGvu4d7zoTz83nfz329ENYfe9x5Q==',
```
Comment (abhi-markan) [inline src/ui/server/constants/integrity.ts:2]: Is this `main.js`, if yes does SRI for this also needs to be updated give `main.js` has also been updated?

**Review by abhi-markan — CHANGES_REQUESTED** [review]

File: src/ui/server/constants/integrity.ts (hunk 1 above)
Comment (ttbarnes) [reply src/ui/server/constants/integrity.ts:2]: I don't believe it does @abhi-markan , `main.js` purely imports SCSS, there's no actual client side JS used in this file.

I think we should remove this integrity hash instead

**Review by abhi-markan — APPROVED** [review]

File: src/ui/public/js/govukFrontend.js.LICENSE.txt [hunk 2]
```diff
@@ -77,6 +77,12 @@
  * @preserve
  */
 
+/**
```
Comment (Zainzzkk) [inline src/ui/public/js/govukFrontend.js.LICENSE.txt:80]: was this done automatically?

**Review by Zainzzkk — APPROVED** [review]

**Review by abhi-markan — APPROVED** [review]

File: src/ui/public/js/govukFrontend.js.LICENSE.txt (hunk 2 above)
Comment (ttbarnes) [reply src/ui/public/js/govukFrontend.js.LICENSE.txt:80]: Yes, it's populated by GOV design library.

**Review by Zainzzkk — APPROVED** [review]

**Pull request merged**